app/posts/register-trainee-teachers/2024-06-25-the-relationship-between-recruitment-cycles-and-academic-years.md

```diff
@@ -0,0 +1,70 @@
+      href: https://www.gov.uk/government/publications/initial-teacher-training-criteria
+    - text: Managing and communicating rollover in 2023
+      href: /publish-teacher-training-courses/managing-and-communicating-rollover-2023/
+---
+
+Postgraduate teacher training has:
+
+- recruitment cycles
+- academic years
+
+A recruitment cycle precedes an academic year. For example, recruitment cycle 2023 to 2024 recruits trainees for the academic year 2024 to 2025.
+
+[![Diagram showing the relationship between recruitment cycles and academic years, including important events like rollover](recruitment-cycles-and-academic-years.png "Diagram showing the relationship between recruitment cycles and academic years (select image to view larger version)")](recruitment-cycles-and-academic-years.png)
+
+## Recruitment cycles
+
+Postgraduate teacher training follows a ‘recruitment cycle’ from October to September.
+
+All training providers must follow the recruitment cycle dates. This applies even if they do not use Publish teacher training courses (Publish) and Manage teacher training applications (Manage).
+
+A recruitment cycle has several important dates. For example, for the 2024 to 2025 recruitment cycle:
+
+|Date and time|What happens|
+| --- | --- |
+|1 October 2024 at 9&nbsp;am |Candidates can find courses for the 2024 to 2025 recruitment cycle on GOV.UK.|
+|8 October 2024 at 9&nbsp;am |Start of 2024 to 2025 recruitment cycle. Candidates can apply for courses.|
+|16 September 2025 at 6&nbsp;pm |The last day for candidates to apply for courses.|
+|24 September 2025 at 11:59&nbsp;pm |End of 2024 to 2025 recruitment cycle. Applications awaiting decisions are rejected.|
+
+### Rollover
+
+Rollover is a period of time towards the end of a recruitment cycle where we copy ‘open’ and ‘closed’ courses from the current recruitment cycle into the new recruitment cycle so that training providers can begin preparing for the next recruitment cycle.
+
+The rollover period typically runs from July until Find postgraduate teacher training (Find) opens in October.
+
+## Academic years
+
+Academic years are determined by the course start date.
+
+|If the course starts between:|The academic year is:|
+| --- | --- |
+|1 August 2024 to 31 July 2025|2024 to 2025|
+|1 August 2023 to 31 July 2024|2023 to 2024|
+|1 August 2022 to 31 July 2023|2022 to 2023|
+
+For most students, each academic year runs from 1 September to 31 August, but there are 4 official start dates in each academic year, each lasting 12 months.
+
+|If the course starts between:|The academic year will run from:|
+| --- | --- |
+|1 August and 31 December|1 September to 31 August|
+|1 January and 31 March|1 January to 31 December|
+|1 April and 30 June|1 April to 31 March|
+|1 July and 31 July|1 July to 30 June|
+
+For postgraduate teaching apprenticeships (PGTA), the academic year may start as early as 1 June.
```

Comment:
Where was this sourced from?

Can we have an example of expected academic year, ie 1 June 2023 is it 2022 to 2023 or 2023 to 204

Reply:
The `1 June` date was sourced from [Postgraduate teaching apprenticeship funding manual: 2024 to 2025 academic year](https://www.gov.uk/government/publications/postgraduate-teaching-apprenticeships-funding-manual/postgraduate-teaching-apprenticeship-funding-manual-2024-to-2025-academic-year#grant-funding-rates). @olioldfield might know what academic year these courses will fall within.

Reply:
For PGTA, courses from1 June 2023 fall into the next academic year so 2023-24. This is because they are a minimum of 12 months long, unlike other postgrad courses. This is important for the salary grant funding that these courses.